package.json

```diff
@@ -8,8 +8,8 @@
   "homepage": "https://github.com/alner/qsSimpleKPI",
   "bugs": "https://github.com/alner/qsSimpleKPI/issues",
   "scripts": {
-    "prod": "set NODE_ENV=production && gulp production",
-    "dev": "set NODE_ENV=development && gulp development",
+    "prod": "export NODE_ENV=production && gulp production",
```

Comment:
Pretty sure this won't work on windows, an alternative for setting it across platforms is to use cross-env (https://www.npmjs.com/package/cross-env)